### ISSUE TITLE
Index is incomplete

### ISSUE THREAD

**Issue by srid** [hmemcpy/milewski-ctfp-pdf#212]
The _Index_ provided at the back of the book, which is a mere 2 pages, seems incomplete. 

For example I can locate neither `epimorphism` nor `monomorphism`, which Bartosz discusses in [one of his video lectures](https://youtu.be/NcT7CGPICzo), in it.

**Comment by BartoszMilewski** [comment]
Yes, it would be nice if somebody volunteered to create a more comprehensive index. It's not an easy job though.

**Comment by hmemcpy** [comment]
Yes, some help would be greatly appreciated! If it helps, the `\newterm` macro automatically adds the term to index!

**Comment by 0rzech** [comment]
I want to read the book, so I could place `\newterm`s along the way on my branch. Both reading and marking will be done in my free time, so no deadlines guaranteed, ha ha! :)

I can see that there's some line length limit in tex files. Personally, I prefer to stick to "1 sentence - 1 line" rule, because that way adding/removing words and macros doesn't rearrange everything that follows. So, would it be fine if I just ignored the line-length limit when adding `\newterm`s? Additional benefit is that diffs will be simpler that way.
